react-frontend/src/redux-store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore } from "redux";
+import rootReducer from "reducers/rootReducer";
```

Comment:
For consistency, follow this import order:

- core/important imports from third party modules
- other third party imports
- line break
- import custom modules
- line break
- import CSS modules